lib/reservation_generator_helper.rb

```diff
@@ -0,0 +1,96 @@
+module ReservationGeneratorHelper
```

Comment:
The move was mostly motivated by trying to keep the files less than 100 lines, and these methods are clearly a separate group from the other methods necessary for generation, but I could put them back